### DISCUSSION TITLE
data format in select_features

### DISCUSSION THREAD

**Discussion by kkckk1110** [blue-yonder/tsfresh#1065] [Q&A]
I have a problem with the data format for the select_features function. the document said the data should be index, feature 1, .., feature N. What should be the index? I have multiple time series distinguished by id and date, how should I prepare my data for the function?

**Comment by nils-braun** [comment]
Hi @kkckk1110 - sorry for the super late response (I just missed your question). The index is the normal pandas index. It is used to relate the "X" and "y" of the data. For tsfresh, it does not matter at all what exactly you put into the index, as long as the "X" an the "y" have the same index (and we can relate the two)
